roles/forkliftcontroller/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-app_name: "{{ lookup('env', 'APP_NAME') | default('forklift') }}"
-app_namespace: "{{ lookup( 'env', 'WATCH_NAMESPACE') | default('openshift-migration') }}"
+app_name: "{{ lookup('env', 'APP_NAME') or 'forklift' }}"
+app_namespace: "{{ lookup( 'env', 'WATCH_NAMESPACE') or 'openshift-migration' }}"
```

Comment:
Any specific reason to use `or` instead of `default` filter? AFAIU, `or` is used in boolean expression, so I would expect the result to be a boolean. Maybe it was introduced by Ansible as another way to fallback to a default value, but the `default` filter is more explicit.

Reply:
The default filter is more explicit but has issues with the lookup plugin, in fact there is another way to make it work in this fashion : 
```
{{ lookup('env', 'APP_NAME') | d('forklift',true) }}"

Reply:
Thanks. That's my TIL :)